### DISCUSSION TITLE
Explanation of how to connect water tank to droplet controller

### DISCUSSION THREAD

**Discussion by stefancvetkovic** [PricelessToolkit/Droplet#9] [Q&A] [answered]
Hello,
I love your project, but Im not surehow to conenct my water tank and water pump as input to droplet controller.
Thanks!

**Comment by PricelessToolkit** [comment]
Hi, If you mean this type of pump "https://s.click.aliexpress.com/e/_DC1usdz" they have water input and output ports. Just throw the inlet hose into the water tank "height difference should not be very large", and the output hose goes to the plants.
droplet has 5 outputs for pumps each capable of outputting "5V 400mA" so if you want to connect other pumps, pay attention pump's power consumption.

**Reply by stefancvetkovic** [reply]
Thanks for the quick response!
When I go to your shop, I cant choose main board + expansion board? 
Do you have this board in stock or? 
https://i.imgur.com/cXvRlLY.png

**Reply by PricelessToolkit** [reply, marked as answer]
Unfortunately, I don't have an expansion board in stock. and I can't say when it will be in stock again.
